src/Controllers/FrontEnd/CoursesPageController.php

```diff
@@ -21,14 +21,18 @@ public function __construct(PhpRenderer $renderer, CourseModel $courseModel)
 
     /**
      * Checks for logged-in status,
-     * gets courses categories from DB
+     * gets courses with future end dates categories from DB
      * and returns rendered landing screen for Courses page
      */
     public function __invoke(Request $request, Response $response, array $args): Response
     {
         if ($_SESSION['loggedIn'] === true) {
-            $courses = $this->courseModel->getAllCourses();
-            $args['courses'] = $courses;
+            $ongoingCourses = $this->courseModel->getOngoingCourses();
+            $args['ongoingCourses'] = $ongoingCourses;
```

Comment:
this can be done in 1 line, why make the variable to immediately put it into an array, just lose the variable

Reply:
and same on all model methods